package/kernel/mac80211/patches/985-rt2x00-add-rxiq-calibration.patch

```diff
@@ -0,0 +1,417 @@
+--- a/drivers/net/wireless/ralink/rt2x00/rt2800lib.c
++++ b/drivers/net/wireless/ralink/rt2x00/rt2800lib.c
+@@ -7923,6 +7923,386 @@ void rt2800_rxdcoc_calibration(struct rt
+ }
+ EXPORT_SYMBOL_GPL(rt2800_rxdcoc_calibration);
+ 
++static u32 rt2800_do_sqrt_accumulation(u32 si) {
++	u32 root, root_pre, bit;
++	char i;
++	bit = 1 << 15;
++	root = 0;
++	for (i = 15; i >= 0; i = i - 1) {
++		root_pre = root + bit;
++		if ((root_pre*root_pre) <= si)
++			root = root_pre;
++		bit = bit >> 1;
++	}
++
++	return root;
++}
++EXPORT_SYMBOL_GPL(rt2800_do_sqrt_accumulation);
```

Comment:
You declare `rt2800_do_sqrt_accumulation` as static above which contradicts the export. Besides that, what would be the external user of that function?